src/emulator/gui/src/settings_dialog.cpp

```diff
@@ -49,19 +41,66 @@ bool change_pref_location(const std::string &input_path, const std::string &curr
 }
 
 void draw_settings_dialog(HostState &host) {
+    static ImGuiTextFilter search_bar;
```

Comment:
This should probably be in the GUI state structure rather than static.

Reply:
hmm i see, just other dev have said me put all adding line here